fdbclient/include/fdbclient/TenantManagement.actor.h

```diff
@@ -492,6 +493,35 @@ Future<Void> configureTenantTransaction(Transaction tr,
 	return Void();
 }
 
+ACTOR template <class Transaction>
+Future<Void> changeLockState(Transaction* tr, int64_t tenant, TenantLockState desiredLockState, UID lockID) {
```

Comment:
It's perhaps not ideal, but the other functions here specify the parameter without the pointer (i.e. `Transaction tr`) so that they can accept both `Reference<Transaction> and `Transaction*`. If you decide to do the same, the caller of this function in TenantCommands could pass in `tr` rather than `tr.getPtr()`.